src/App.jsx

```diff
@@ -1,3 +1,51 @@
+import React from "react";
```

Comment:
great structure, I can follow this well, descriptive component names are also great!